### PR TITLE
Make Kernel A/B updates (as part of rootfs) default behavior for meta-rauc-raspberrypi

### PR DESCRIPTION
**Pull request by AtanasBunchev**
I have successfully done a kernel update using a RAUC update bundle on Raspberry Pi 5 with these patches and the instructions from the README.rst.

### CODE REVIEW

**Review by leon-anavi — APPROVED** [review]

File: meta-rauc-raspberrypi/recipes-bsp/rpi-u-boot-scr/files/boot.cmd.in [hunk 1]
```diff
@@ -41,6 +41,6 @@ else
   reset
 fi
 
-fatload mmc 0:1 ${kernel_addr_r} @@KERNEL_IMAGETYPE@@
+load ${BOOT_DEV} ${kernel_addr_R} boot/@@KERNEL_IMAGETYPE@@
```
Comment (ejoerns) [inline meta-rauc-raspberrypi/recipes-bsp/rpi-u-boot-scr/files/boot.cmd.in:44]: Is the change to `kernel_addr_R` made intentionally?

Comment (AtanasBunchev) [reply meta-rauc-raspberrypi/recipes-bsp/rpi-u-boot-scr/files/boot.cmd.in]: That was definitely a typo, thanks for catching it.

**Comment by AtanasBunchev** [comment]
I wanted to make sure to have tested it properly, so I used the 'Draft' status to make sure it doesn't get merged too early in case something doesn't work quite right.
Thanks again for the review :)

**Comment by ejoerns** [comment]
@leon-anavi Maybe a good chance before merging this: I recently enforced the merge policy to "rebase" for `meta-rauc` and other layers I maintain.
Do you (or anyone else) have an opinion on **merge vs. rebase** for this layer?
Normally, I'd argue for merges (as we do for *rauc*) since they give a clearer context and mark the source of the PR.
However, for other meta layers, the history is often linear and also here most PRs are single commits and patches are more or less independent from each other. Especially for maintaining different stable branches in parallel, I find linear history a bit more readable.

**Comment by leon-anavi** [comment]
> @leon-anavi Maybe a good chance before merging this: I recently enforced the merge policy to "rebase" for `meta-rauc` and other layers I maintain. Do you (or anyone else) have an opinion on **merge vs. rebase** for this layer? 

I don't have a hard opinion on this topic. Would you like to switch to "rebase"  merge policy for `meta-rauc-community`?

**Comment by ejoerns** [comment]
> > @leon-anavi Maybe a good chance before merging this: I recently enforced the merge policy to "rebase" for `meta-rauc` and other layers I maintain. Do you (or anyone else) have an opinion on **merge vs. rebase** for this layer?
> 
> I don't have a hard opinion on this topic. Would you like to switch to "rebase" merge policy for `meta-rauc-community`?

Ok, let's give it a try. :+1:

**Comment by leon-anavi** [comment]
> > > @leon-anavi Maybe a good chance before merging this: I recently enforced the merge policy to "rebase" for `meta-rauc` and other layers I maintain. Do you (or anyone else) have an opinion on **merge vs. rebase** for this layer?
> > 
> > 
> > I don't have a hard opinion on this topic. Would you like to switch to "rebase" merge policy for `meta-rauc-community`?
> 
> Ok, let's give it a try. 👍

OK.

I don't have access to the "Settings" of this git repository anymore. Do you? Could you please enforce the default merge policy to "rebase" as for meta-rauc?

**Comment by ejoerns** [comment]
> I don't have access to the "Settings" of this git repository anymore. Do you? Could you please enforce the default merge policy to "rebase" as for meta-rauc?

Should be done already :wink:

**Pull request merged**